### ISSUE TITLE
linter: jest/prefer-ending-with-an-expect false positive with vi.mock

### ISSUE THREAD

**Issue by cexbrayat** [oxc-project/oxc#21413]
### What version of Oxlint are you using?

1.60.0

### What command did you run?

oxlint -c .oxlintrc.json

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```jsonc
{                                                                                                                     
  "plugins": ["jest"],                                       
  "rules": {                                                                                                          
    "jest/prefer-ending-with-an-expect": "error"                                                                      
  }
}
```

### What happened?

The command throws an error for vi.mock calls

```
2 |
      3 | vi.mock(import('foo'), () => vi.fn());
        : ^^^^^^^
        `----
       help: Add an `expect` or assertion call as the last statement in the test block.
```
